turbo.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "pipeline": {
+    "release": {},
+    "build": {},
+    "compile": {
+      "dependsOn": ["^compile"]
+    },
```

Comment:
compile depends on not compile?

Reply:
> compile depends on not compile?

This threw me for a loop as well. The carrot means for all my dependencies. So a given workspace requires all its dependencies to be compiled first.

https://turborepo.org/docs/core-concepts/running-tasks#in-a-different-workspace